tests/unit/test_commands.py

```diff
@@ -233,57 +228,10 @@ def test_parse_args_quiet_verbose_exclusive(args):
                 backup=True,
                 force=True,
                 machines=None,
-                hostnames=None,
                 availability_zones={"1", "2", "3"},
                 **{"upgrade_group": "data-plane"}
             ),
         ),
-        (
-            ["plan", "data-plane", "--hostname=1", "-n=2,3"],
-            CLIargs(
-                command="plan",
-                model_name=None,
-                verbosity=0,
-                quiet=False,
-                backup=True,
-                force=False,
-                machines=None,
-                hostnames={"1", "2", "3"},
-                availability_zones=None,
-                **{"upgrade_group": "data-plane"}
-            ),
-        ),
-        (
-            ["plan", "data-plane", "--hostname=1", "--force", "-n=2,3"],
-            CLIargs(
-                command="plan",
-                model_name=None,
-                verbosity=0,
-                quiet=False,
-                backup=True,
-                force=True,
-                machines=None,
-                hostnames={"1", "2", "3"},
-                availability_zones=None,
-                **{"upgrade_group": "data-plane"}
-            ),
-        ),
-        (
-            # repetitive machine 3
-            ["plan", "data-plane", "--hostname=1,2,3", "--force", "-n=3,4"],
```

Comment:
I think we can keep this test and just change `hostname`to `machine`. This test is also checking that no repetitive elements appears after the parding